### PR TITLE
Meat Golems

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
add: Golems can now build golem shells out of meat. This is considered an abomination in golem culture and should never be attempted.
/:cl:

@RemieRichards requested this.

Basically makes a human instead of a golem. "Meat Golems" cannot be servants, even if made from a servant golem shell, and cannot be transferred to from another golem.

### CODE REVIEW

**Comment by PraiseRatvar** [comment]
should make them a bloated human instead

**Comment by Armhulen** [comment]
NOT COOL, BRO.

**Comment by Armhulen** [comment]
just use the bloated skin and make them a REAL meat golem

**Comment by ExcessiveUseOfCobblestone** [comment]
Second Life: The Ghostrole

**Comment by FantasticFwoosh** [comment]
I laughed but this is a waste of resources to get back into the round fast.

**Comment by Kromgar** [comment]
Yeah Bloated Humans so the ai can distinguish between human and nonhuman.

**Comment by ExcessiveUseOfCobblestone** [comment]
or better yet add actual flesh golems instead of clearance humans.

**Comment by Ratvar** [comment]
Hm. Make them leave blood drops, spread blood on hit, remove space/bee/chem resistance, but leave inability to wear clothes, make them gibbable with convinient sharp items, give them toggleable cult robe speed boost that also deals 0.2 damage to every body part per walked tile? Meatboy style. Or just leave them memey humans

**Comment by fludd12** [comment]
@XDTM How about, instead of meat, it's a hide golem? That way, you don't need to snowflake in the meat counter, and just use sheets of human hide. Also stops people from being able to mass produce the fucking things.

**Comment by KorPhaeron** [comment]
Was expecting gross raw meat blobs from the title, dissapointed

**Comment by GunHog** [comment]
Agreed with Kor. It should not be just a straight-up respawn! I WANT A MEAT SLAVE!!

**Comment by Armhulen** [comment]
basically what kor said, free respawns go elsewhere!

**Comment by Armhulen** [comment]
it should like, be able to eat any item and stuff on a minute cooldown like they do with goon

**Comment by RemieRichards** [comment]
>everyone in this PR
>absolute LACK of appreciation for jokes

you people sicken me

**Comment by DoctoreMad** [comment]
I think people were laughing at you yesterday in the IRC.

**Comment by RemieRichards** [comment]
??

**Comment by XDTM** [comment]
Sorry remie, given the feedback i'm going to make actual flesh golems

**Comment by RemieRichards** [comment]
That's fine.
I would like to know what the fuck @DoctoreMad is talking about though.

**Comment by Hatfish** [comment]
What do you get when you slap lizard, podman or xeno meat onto a golem shell?

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -186,6 +186,36 @@
...
+	anchored = FALSE
+	density = FALSE
+	flavour_text = "<font size=3><b>Y</b></font><b>ou are a Free Golem... wait, no. You're not. You're a human. As you realize this, you feel the wave of ancestral golem knowledge falter and stop, \
+	leaving nothing aside from a mild headache. You're left not knowing anything about your past, if you even had one.</b>"
+
+/obj/effect/mob_spawn/human/golem/Initialize(mapload)
+	. = ..()
+	var/area/A = get_area(src)
+	if(!mapload && A)
+		notify_ghosts("\A meat golem shell has been completed in \the [A.name].", source = src, action=NOTIFY_ATTACK, flashwindow = FALSE)
```
Comment (Cyberboss) [inline code/game/objects/structures/ghost_role_spawners.dm]: No need for \A at the start

File: code/modules/ruins/lavaland_ruin_code.dm [hunk 2]
```diff
@@ -110,13 +111,29 @@
...
-				qdel(src)
+		if(istype(O, /obj/item/reagent_containers/food/snacks/meat/slab))
+			qdel(O)
+			meat++
+			if(meat < 10)
+				to_chat(user, "You add [O] to the golem shell. It now has [meat]/10 slabs of meat.")
 			else
-				to_chat(user, "You need at least ten sheets to finish a golem.")
+				to_chat(user, "You finish up the golem shell with [O].[isgolem(user) ? " You feel weird about this." : ""]")
+				new /obj/effect/mob_spawn/human/meatgolem(get_turf(src))
```
Comment (Cyberboss) [inline code/modules/ruins/lavaland_ruin_code.dm]: get_turf(src) -> drop_location()

File: code/modules/ruins/lavaland_ruin_code.dm [hunk 3]
```diff
@@ -110,13 +111,29 @@
...
+
+		if(species)
+			if(meat)
+				to_chat(user, "This golem is being built out of meat. You cannot add other materials to it.")
+				return
+			if(istype(O, /obj/item/stack)) //just to be safe
+				var/obj/item/stack/S = O
+				if(S.use(10))
+					to_chat(user, "You finish up the golem shell with ten sheets of [O].")
+					new shell_type(get_turf(src), species, user)
```
Comment (Cyberboss) [inline code/modules/ruins/lavaland_ruin_code.dm]: ditto

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by DoctoreMad** [comment]
Well I was in the IRC and OH SHIT XTDM WHAT DO YOU MEAN YOU WANT TO DO ACTUAL MEAT GOLEMS? HOLY SHIT.

**Comment by RemieRichards** [comment]
I'm honestly not sure if you're even speaking a human language right now, let alone english.

**Comment by Cyberboss** [comment]
@XDTM

**Comment by XDTM** [comment]
sorry about the inactivity, had some busy days at work

gonna code this during the weekend

**Comment by XDTM** [comment]
Closing in favor of the new PR

**Pull request closed without merging**